### PR TITLE
Python3 - fixes that improve the unit tests

### PR DESCRIPTION
**Pull request by guedou**
These commits fix Python3 related unit tests.

### CODE REVIEW

File: scapy/main.py [hunk 1]
```diff
@@ -176,7 +176,8 @@ def list_contrib(name=None):
         if mod.endswith(".py"):
             mod = mod[:-3]
         desc = { "description":"-", "status":"?", "name":mod }
-        for l in open(f):
+        from io import open
```
Comment (p-l-) [inline scapy/main.py]: Can you move this import statement to the beginning of the file?

Comment (guedou) [reply scapy/main.py]: Done !

File: scapy/main.py [hunk 2]
```diff
@@ -16,6 +16,7 @@
 import logging
 from random import choice
 import types
+from io import open
```
Comment (p-l-) [inline scapy/main.py]: Sorry, can you not overwrite the built-in `open`? Using  `import io` and `io.open`, for example?

Comment (guedou) [reply scapy/main.py]: Sorry for this silly change ... I updated the commit.

File: test/regression.uts [hunk 3]
```diff
@@ -266,10 +266,6 @@ Ether(dst="ff:ff:ff:ff:ff:ff", src="ff:ff:ff:ff:ff:ff", type=0x9000)
 assert _.mysummary() in ['ff:ff:ff:ff:ff:ff > ff:ff:ff:ff:ff:ff (%s)' % loop
                          for loop in ['0x9000', 'LOOP']]
 
-= Test fletcher16_* functions
-assert(fletcher16_checksum(b"\x28\x07") == 22319)
-assert(fletcher16_checkbytes(b"ABCDEF", 2) == "\x89\x67")
```
Comment (gpotter2) [inline test/regression.uts]: Hold on, it seems that simply using
`assert(fletcher16_checkbytes(b"ABCDEF", 2) == b"\x89\x67")`
fixes it

**Comment by guedou** [comment]
The tear is duplicated. The other one already  uses b”” =)

File: test/regression.uts (hunk 3 above)
Comment (gpotter2) [reply test/regression.uts]: Oh, you're right ^^

**Pull request merged**